service/controller/resource/monitoring/ingress/resource.go

```diff
@@ -60,8 +60,10 @@ func getObjectMeta(v interface{}, config Config) (metav1.ObjectMeta, error) {
 	}
 
 	annotations := map[string]string{
-		"nginx.ingress.kubernetes.io/auth-signin": "https://$host/oauth2/start?rd=$escaped_request_uri",
-		"nginx.ingress.kubernetes.io/auth-url":    "https://$host/oauth2/auth",
+		"nginx.ingress.kubernetes.io/auth-signin":   "https://$host/oauth2/start?rd=$escaped_request_uri",
+		"nginx.ingress.kubernetes.io/auth-url":      "https://$host/oauth2/auth",
+		"external-dns.alpha.kubernetes.io/hostname": config.BaseDomain,
```

Comment:
Should this not be added only if `.Values.ingress.externalDNS` is true?

Reply:
yes, for sure - just wanted your confirmation on this :sweat_smile: